Frontend/src/components/TableEditors/Rentals/Rentals.svelte

```diff
@@ -24,6 +47,26 @@
     return isBeforeDay(millis, new Date().getTime());
   }
 
+  async function cellBackgroundColorFunction(col, item) {
```

Comment:
```suggestion
 function cellBackgroundColorFunction(col, item) {
```
You can remove the async here. You only need this if you use "await" inside of a function.

Reply:
I think we should rename the second argument passed to the function to "rental", item is confusing here. I know it has been that way before.

Reply:
And since the function not only returns background colors we should rename it to something else, maybe "cellStyle"?